ang/civicase/ActivitiesCalendar.js

```diff
@@ -20,13 +21,32 @@
      * @param {Object} element
      */
     function civicaseActivitiesCalendarLink ($scope, element) {
+      var domObserver;
```

Comment:
What is the role of `domObserver`?

Reply:
The dom observer watches for changes to the DOM in order to make adjustments to it. The following manipulations are done:

* splitting the year and month title on the calendar into separated spans elements so they can be styled differently. There's no CSS rule to match words so I had to resort to this method.
* The calendar arrows use Font Awesome, but they should be Material Icons so they are changed manually. As far as I investigated there's no way to change to material icons through CSS changes so I had to manipulate the DOM instead.
* The current week day is nor marked in the calendar so a new class had to be provided for it.

The DOM Mutation Observer was also useful because when the activity calendar directive's link function is provided the UI Bootstrap's datepicker HTML element is not ready so the DOM Observer waits for it, and also triggers every time the calendar is redrawn.